FAQ.md

```diff
@@ -51,97 +55,100 @@ types of programming languages.
+Yes. You can publish your own source code under a permissive license compatible with
+the GPL, such as BSD, MIT, or Apache license. However, the binary or the package
+containing Slint needs to be licensed under GPL. For more info, you can check the following GPL FAQs:
 
-- The Ambassador option is a free license that can be provided in lieu of helping
-us promote Slint. Apart from the standard inclusiosn, unlimited User-Seats, updates and unlimited distribution 
-of Slint based applications on embedded devices are included.
+- [https://www.gnu.org/licenses/gpl-faq.en.html#LinkingWithGPL](https://www.gnu.org/licenses/gpl-faq.en.html#LinkingWithGPL)
+- [https://www.gnu.org/licenses/gpl-faq.en.html#IfLibraryIsGPL](https://www.gnu.org/licenses/gpl-faq.en.html#IfLibraryIsGPL)
 
-- With the Flex option, you can purchase the required number of User-Seats needed to
-develop your applications with Slint. In addition to the standard inclusions, updates are included as long 
-as the subscription is active.
+### What are the different proprietary licensing options?
 
-- The Buyout option includes unlimited User-Seats, updates and 30 hours of [premium support](https://slint-ui.com/support/slint_premium_support_service_agreement) in addition to the standard inclusions. 
-Slint based applications can be distributed on embedded devices as per purchased distribution license pack.
+|   | Ambassador  | Flex  | Buyout  |
+|--- |:---: |:---: |:---: |
+| Price  | Free of charge  | from EUR 59 /user/month  | from EUR 5900  |
+| Type of license  | Perpetual  | Perpetual  | Perpetual  |
+| Users  | Unlimited  | As per license  | Unlimited  |
+| Distribution on Desktop  | Unlimited  | Unlimited  | Unlimited  |
+| Distribution on Embedded  | Unlimited  | As per purchased volume  | As per purchased volume  |
+| Support  | Standard  | Standard  | Standard + 30 hours of Premium Support  |
+| Updates  | Included  | Included  | Included  |
+| Additional Obligations  | (1) consent to showcase the application,  (2) attribution of Slint and  (3) feedback to improve Slint.  | None  | None  |
 
-### Is the Ambassador really for free? What is the catch?
+The terms and conditions of the proprietary license is available [here](LICENSES/LicenseRef-Slint-commercial.md).
 
-The Ambassador is really for free! The only 'catch' is to help us promote Slint in the following ways:
+### What does perpetual mean?
 
-- Showcase: You grant us the right to use your name, logo and your Products in our marketing materials,
+The perpetual right allows you to use the version(s) of Slint, provided under the
+proprietary license, for ever.
 
-- Spread the word: You will include reasonably visible notices in the documentation, marketing materials 
-and social media channels related to your Products that such were built with Slint,
+### What are the different support options?
```

Comment:
Same, i'd just link to the website.

Reply:
Since we do not have a comparison table, removed it for now and instead mentioned - Standard support and premium support.